### PR TITLE
Handle Multiple Sequences 

### PR DESCRIPTION
**Pull request by vivbak**

### CODE REVIEW

**Comment by vivbak** [comment]
Pending solution here https://github.com/populationgenomics/sample-metadata/issues/321

File: update_sample_status/main.py [hunk 1]
```diff
@@ -106,6 +142,6 @@ def update_sample_status(request):  # pylint: disable=R1710
 
     update_airtable(project_config, sample, status)
 
-    update_sm(project, sample, status, batch)
+    update_sm(project, sample, status, batch, eseqid)
```
Comment (illusional) [inline update_sample_status/main.py:145]: Is `eseqid` a map of `{name_of_provider: external_id}`, eg: `{"kccg": "EX01"}`?
That's what metamist expects, and what's annotated in `update_sm`, but the variable names make me less confident.

**Review by illusional — APPROVED** [review]
Change looks good except for external_sequence_id format.

File: update_sample_status/main.py (hunk 1 above)
Comment (vivbak) [reply update_sample_status/main.py:145]: Yeah the function kind of assumes that eseqid will be provided as a dict in the request. But I think it's probably better to just take eseqid as a string in the request and create the dict within this script.

Comment (vivbak) [reply update_sample_status/main.py:145]: Will amend, thanks :)

**Comment by vivbak** [comment]
Current test run broke on 
`exceptionMessage: "'SequenceApi' object has no attribute 'get_sequence_by_external_id'"` 
So I believe this is pending the changes here https://github.com/populationgenomics/sample-metadata/pull/325

**Comment by vivbak** [comment]
Ah look at that, I actually made an issue for this https://github.com/populationgenomics/sample-metadata/issues/284

**Pull request merged**